sql_export/tests/__init__.py

```diff
@@ -0,0 +1,2 @@
+# -*- coding: utf-8 -*-
+from . import test_sql_query
```

Comment:
Well, I can't see if it ran the tests.
Strange, because in localhost, my odoo did run the tests without any problem.